### PR TITLE
🧹 Documentation changes for some scaffolder packages

### PR DESCRIPTION
**Pull request by benjdlambert**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend-module-rails/README.md [hunk 1]
```diff
@@ -183,44 +183,45 @@ spec:
...
+      if: ${{ parameters.dryRun !== true }}
       name: Publish
       action: publish:github
       input:
-        allowedHosts: ['github.com']
-        description: 'This is {{ parameters.name }}'
-        repoUrl: '{{ parameters.repoUrl }}'
+        allowedHosts:
+          - github.com
+        description: This is {{ parameters.name }}
```
Comment (freben) [inline plugins/scaffolder-backend-module-rails/README.md:201]: missing dollar sign

File: plugins/scaffolder-backend-module-cookiecutter/README.md [hunk 2]
```diff
@@ -109,41 +109,42 @@ spec:
...
+      if: ${{ parameters.dryRun !== true }}
       name: Publish
       action: publish:github
       input:
-        allowedHosts: ['github.com']
-        description: 'This is {{ parameters.name }}'
-        repoUrl: '{{ parameters.repoUrl }}'
+        allowedHosts:
+          - github.com
+        description: This is {{ parameters.name }}
```
Comment (freben) [inline plugins/scaffolder-backend-module-cookiecutter/README.md:124]: missing $

**Review by freben — APPROVED** [review]
A few small notes, but pre-approving

File: plugins/scaffolder-backend/sample-templates/bitbucket-demo/template.yaml [hunk 3]
```diff
@@ -63,16 +63,16 @@ spec:
...
       input:
-        repoContentsUrl: '{{ steps.publish.output.repoContentsUrl }}'
+        repoContentsUrl: ${{ steps.publish.output.repoContentsUrl }}
         catalogInfoPath: '/catalog-info.yaml'
 
   output:
-    remoteUrl: '{{ steps.publish.output.remoteUrl }}'
-    entityRef: '{{ steps.register.output.entityRef }}'
+    remoteUrl: ${{ steps.publish.output.remoteUrl }}
+    entityRef: ${{ steps.register.output.entityRef }}
```
[the remark this answers is not in the document]
Comment (benjdlambert) [reply plugins/scaffolder-backend/sample-templates/bitbucket-demo/template.yaml:78]: Ah sorry, missed that had been removed. Will fix it up.

**Pull request merged**